### DISCUSSION TITLE
Please, build in consistent functionality for Prev/Next buttons in a sequential serie of single articles

### DISCUSSION THREAD

**Discussion by sthopd** [joomla/joomla-cms#43317] [Q&A] [answered]
Hello,
Is Joomla only meant for publishing newspapers, magazines and blogs?
I want to write an online educational book by means of Joomla 5 as publication medium.
The book consists of hundreds of pages (on paper) that are not always written in the correct order of the ongoing story.
The book will be created by 'progressive insight' and is continuously adjusted/corrected (online digital reconstruction).
This means that new webpages (Single Articles) are frequently inserted between existing pages in Joomla-FrontEnd.
Is this digitally possible in Joomla5 Frontend?
Joomla has the 2 separate, simple buttons: 1. on the left side of (some?) webpages (Single Article): '<Prev' 
2. and on the right side of (some?) webpages: 'Next>'.

The current Joomla-functionality of the Prev/Next buttons is too simple, because it only takes into account the sequence of 'publish_up' or 'created date'. At least, that's how it seems.
Is it possible that Joomla developers extend the Settings-functionality in such a way that the webmaster or author can set and edit the sequential order of Prev/Next hops between story-pages? 
The author needs to adjust the sequence of existing articles (Previous page/Next page) to fit the reading-order of an ongoing story, to make it comprehensible and lively for online visitors or students.
For example: the way how it is done in Moodle LMS.
In other words:
Is it possible to program (in code) an expansion of the existing functionality for the Prev/Next buttons (as an extra settings-option in the backend/admin-dashboard) for example:

a) Couple the sequential articles read-order of Previous/Next buttons to the sequential item-order in the (sub)menu designated by the author?
or:
b) Couple the sequential articles read-order of Previous/Next buttons to the sequential item-order in the designated (sub)category?

Regards,
sthopd

Joomla 5.0.3
Template: Cassiopeia (bare).

**Comment by brianteeman** [comment]
I belive what you are asking for is as shown in this video ?


https://github.com/joomla/joomla-cms/assets/1296369/a12c70f7-05d5-4387-958d-eb3be0d05378

**Reply by sthopd** [reply]
Hello Brian,
Thank you for the video-link which explains how I can use the ordering of the articles with the same Category, to adjust the sequence of page-jumps when clicking on the Prev/Next button. 
That was helpful.
In this video, I also see that the words in the button '<Prev' and 'Next>' are replaced by the name or title of the previous or next article.
In the admin dashboard of J5 I cannot find any setting/option that changes the generic words 'Prev' or 'Next' into a page title.
How do I make the Prev or Next-button display the name of the article to show?
Regards,
Sthopd

**Reply by drmenzelit** [reply]
Go to articles -> options and you will find the settings
![grafik](https://github.com/joomla/joomla-cms/assets/9153168/fe204dfe-5784-42cf-8c82-4b88bb300528)

![grafik](https://github.com/joomla/joomla-cms/assets/9153168/c0583153-b2c7-4383-a10c-7311b9c4573b)

**Reply by sthopd** [reply]
Hello Viviana,
I think your answer does not apply to my question about replacing the words 'Prev' en 'Next' by the name/title of the desired page (either previous page or next page in the reading sequence). I do not understand what 'Read More' link has to do with a long reading sequence of page 1, page 2, page 3 ... page 103 etc.
Regards,
Sthopd

**Reply by brianteeman** [reply, marked as answer]
@drmenzelit was confused. The correct option as shown in the video to use the article title instead as prev/next is

![image](https://github.com/joomla/joomla-cms/assets/1296369/ed196845-bbaa-4ebb-a768-48b116aea874)

![image](https://github.com/joomla/joomla-cms/assets/1296369/23ffc535-e1d5-467f-9818-40e5e6ddc457)

**Reply by sthopd** [reply]
Hello Brian,
Thank you for your answer. It worked.
Regards,
Sthopd

**Reply by drmenzelit** [reply]
🙈 that happens when one try to answer something without enough coffee ...

**Comment by rdeutz** [comment]
Already existing fuctionality
